research/delf/delf/python/training/README.md

```diff
@@ -225,8 +264,18 @@ global features.
 python3 model/export_local_and_global_model.py \
   --ckpt_path=gldv2_training/delf_weights \
   --export_path=gldv2_model_local_and_global \
+  --delg_global_features
+```
+
+If the model was trained with the autoencoder described in the DELG paper, use
+the following export command instead:
+
+```
+python3 model/export_global_model.py \
+  --ckpt_path=gldv2_training/delf_weights \
+  --export_path=gldv2_model_global \
   --delg_global_features \
-  --block3_strides
```

Comment:
same as above

Reply:
If I understand correctly no change is require here, right?